### PR TITLE
[Feature] Add IC11 (Born-digital Images) Data Converter

### PR DESCRIPTION
**Pull request by xinke-wang**
## Motivation

Add data converter for [ICDAR 2011 (Born-digital images)](https://rrc.cvc.uab.es/?ch=1)

## Use cases (Optional)

Please refer to docs for data preparation scripts. 

**FOLLOWING OUTDATED FOLLOWING OUTDATED FOLLOWING OUTDATED FOLLOWING OUTDATED**
### Detection ICDAR 2011 (Born-Digital Images)
- Step1: Download `Challenge1_Training_Task12_Images.zip`, `Challenge1_Training_Task1_GT.zip`, `Challenge1_Test_Task12_Images.zip`, and `Challenge1_Test_Task1_GT.zip` from [homepage](https://rrc.cvc.uab.es/?ch=1&com=downloads) `Task 1.1: Text Localization (2013 edition)`.

```bash
mkdir icdar2011 && cd icdar2011
mkdir imgs && mkdir annotations

# For images
unzip Challenge1_Training_Task12_Images.zip -d imgs/training
unzip Challenge1_Test_Task12_Images.zip -d imgs/test
# For annotations
unzip Challenge1_Training_Task1_GT.zip -d annotations/training
unzip Challenge1_Test_Task1_GT.zip -d annotations/test

rm Challenge1_Training_Task12_Images.zip && rm Challenge1_Test_Task12_Images.zip && rm Challenge1_Training_Task1_GT.zip && rm Challenge1_Test_Task1_GT.zip
```
- Step 2: Generate `instances_training.json` and `instances_test.json` with the following command:

```bash
python tools/data/textdet/ic11_converter.py PATH/TO/icdar2011 --nproc 4
```

### Recognition ICDAR 2011 (Born-Digital Images)
- Step1: Download `Challenge1_Training_Task3_Images_GT.zip`, `Challenge1_Test_Task3_Images.zip`, and `Challenge1_Test_Task3_GT.txt` from [homepage](https://rrc.cvc.uab.es/?ch=1&com=downloads) `Task 1.3: Word Recognition (2013 edition)`.

```bash
mkdir icdar2011 && cd icdar2011
mkdir annotations

# For images
unzip Challenge1_Training_Task3_Images_GT.zip -d train
unzip Challenge1_Test_Task3_Images.zip -d test
# For annotations
mv Challenge1_Test_Task3_GT.txt annotations && mv train/gt.txt annotations/Challenge1_Train_Task3_GT.txt
```

- Step2: Convert original annotations to `train_label.txt` and `test_label.txt` with the following command:

```bash
python tools/data/textrecog/ic11_converter.py PATH/TO/icdar2011
```

### CODE REVIEW

File: tools/data/textrecog/ic11_converter.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            lines.append(
+                json.dumps({
+                    'filename':
+                    f'{osp.basename(dst_image_root)}/{dst_img_name}',
+                    'text': word
+                }))
+        else:
+            raise NotImplementedError
+
+    list_to_file(osp.join(root_path, f'{split}_label.txt'), lines)
```
Comment (gaotongxiao) [inline tools/data/textrecog/ic11_converter.py:59]: Filename extension should match the foramt

File: docs/en/datasets/det.md [hunk 2]
```diff
@@ -54,6 +59,8 @@ The structure of the text detection dataset directory is organized as follows.
...
+|         -         |                                                                                                                                                                                                                                                                                |
+|    ICDAR 2011     |                                                                                                                    [homepage](https://rrc.cvc.uab.es/?ch=1)                                                                                                                    |                                                                                                              -                                                                                                               |                                              -                                               |                                                                                                |
```
Comment (Mountchicken) [inline docs/en/datasets/det.md:63]: Something wrong here

Comment (xinke-wang) [reply docs/en/datasets/det.md]: fixed

**Review by Mountchicken — APPROVED** [review]
Tested on sproject, everything works fine

**Review by gaotongxiao — APPROVED** [review]

**Pull request merged**